<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>图像配准（基于Opencv_Cuda）</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="alignImgLayout">
    <item>
     <widget class="QGraphicsView" name="alignImgView"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonUploadAlignImg">
      <property name="text">
       <string>上传待配准图片</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="refImgLayout">
    <item>
     <widget class="QGraphicsView" name="refImgView"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonUploadRefImg">
      <property name="text">
       <string>上传基准图片</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonShowAlignedImg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示配准结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>匹配特征点</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>171</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="alignedImgLayout">
    <item>
     <widget class="QGraphicsView" name="alignedImgView"/>
    </item>
    <item>
     <widget class="QPushButton" name="buttonDoloadAlignedImg">
      <property name="text">
       <string>下载配准后图片</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
